package.json

```diff
@@ -63,7 +66,10 @@
     "@vue/cli-plugin-unit-mocha": "^4.5.13",
     "@vue/cli-service": "^4.5.13",
     "@vue/test-utils": "^1.2.0",
+    "@yarnpkg/fslib": "^2.5.0-rc.2",
```

Comment:
Leftover from testing?

Reply:
Ah, I think I tried it to workaround the PnP issue I had. Good catch, thanks!!